### PR TITLE
[SoftDeleteable] Allow datetimeMS as a valid type

### PR DESCRIPTION
**Pull request by axio-vincent**

### CODE REVIEW

File: src/SoftDeleteable/Mapping/Validator.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ class Validator
         'datetimetz',
         'datetimetz_immutable',
         'timestamp',
+		'datetimeMS'
```
Comment (phansys) [inline src/SoftDeleteable/Mapping/Validator.php:32]: This type does not seem to be supported by ["doctrine/dbal" 3.x](https://github.com/doctrine/dbal/blob/4534d445068f439d9ec287960ed7525f17b55840/src/Types/Types.php#L12-L36). Where is it declared?

**Comment by phansys** [comment]
Related to #2293.

**Comment by Vin57** [comment]
This is a custom type that we created on our project to allow for specific date time format of SQL Server.

This MR is really bad and should'nt be implemented.

I just fork doctrine extension repository to extend it for myself.

**Comment by phansys** [comment]
@axio-vincent, could you please confirm and close the PR?

**Comment by Vin57** [comment]
/close

> @axio-vincent, could you please confirm and close the PR?

I confirm. I can't close the PR. The option is not available

**Comment by phansys** [comment]
>I confirm. I can't close the PR. The option is not available

You can't close PRs or issues that were created by other users.

**Pull request closed without merging**